extensions/wikia/WallNotifications/tests/WallNotificationEntityTest.php

```diff
@@ -72,50 +68,75 @@ private function getWallMessageMock( array $params ) {
 				'isEdited',
 				'getLastEditSummary',
 				'getMetaTitle',
+				'load'
 			] )
 			->disableOriginalConstructor()
 			->getMock();
+		$wallMessageCounter = 0;
 
-		$wm->expects( $this->once() )
+		$wallMessageMock->expects( $this->at( $wallMessageCounter++ ) )
+			->method( 'load' );
+
+		$wallMessageMock->expects( $this->at( $wallMessageCounter++ ) )
 			->method( 'getWallOwner' )
-			->willReturn( $wallOwner );
-		$wm->expects( $this->once() )
+			->willReturn( $wallOwnerMock );
+
+		$wallMessageMock->expects( $this->at( $wallMessageCounter++ ) )
 			->method( 'getArticleTitle' )
-			->willReturn( $articleTitle );
-		$wm->expects( $this->once() )
-			->method( 'getTitle' )
-			->willReturn( $title );
-		$wm->expects( $this->once() )
-			->method( 'getTopParentObj' )
-			->willReturn( $parentWm );
-		$wm->expects( $this->once() )
+			->willReturn( $articleTitleMock );
+
+		$wallMessageMock->expects( $this->at( $wallMessageCounter++ ) )
 			->method( 'getText' )
 			->willReturn( $params['wallMessage']['text'] );
-		$wm->expects( $this->once() )
+
+		$wallMessageMock->expects( $this->at( $wallMessageCounter++ ) )
+			->method( 'getTitle' )
+			->willReturn( $titleMock );
+
+		$wallMessageMock->expects( $this->at( $wallMessageCounter++ ) )
 			->method( 'getMessagePageUrl' )
 			->willReturn( $params['wallMessage']['messagePageUrl'] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "messagePageUrl" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)